server/scripts/merge-locations.ts

```diff
@@ -4,150 +4,59 @@
-// They're always used in lower-case text where punctuation has been removed.
-// In some cases, the replacements *remove* the abbreviation entirely to enable
-// better loose matching (usually for road types, like "road" vs. "street").
-const ADDRESS_EXPANSIONS = [
-  [/ i /g, " interstate "],
-  [/ i-(\d+) /g, " interstate $1 "],
-  [/ expy /g, " expressway "],
-  [/ fwy /g, " freeway "],
-  [/ hwy /g, " highway "],
-  [/ (u s|us) /g, " "], // Frequently in "U.S. Highway / US Highway"
-  [/ (s r|sr|st rt|state route|state road) /g, " route "],
-  [/ rt /g, " route "],
-  [/ (tpke?|pike) /g, " turnpike "],
-  [/ ft /g, " fort "],
-  [/ mt /g, " mount "],
-  [/ mtn /g, " mountain "],
-  [/ (is|isl|island) /g, " "],
-  [/ n /g, " north "],
-  [/ s /g, " south "],
-  [/ e /g, " east "],
-  [/ w /g, " west "],
-  [/ nw /g, " northwest "],
-  [/ sw /g, " southwest "],
-  [/ ne /g, " northeast "],
-  [/ se /g, " southeast "],
-  [/ ave? /g, " "],
-  [/ avenue? /g, " "],
-  [/ dr /g, " "],
-  [/ drive /g, " "],
-  [/ rd /g, " "],
-  [/ road /g, " "],
-  [/ st /g, " "],
-  [/ street /g, " "],
-  [/ saint /g, " "], // Unfortunately, this gets mixed in with st for street.
-  [/ blvd /g, " "],
-  [/ boulevard /g, " "],
-  [/ ln /g, " "],
-  [/ lane /g, " "],
-  [/ cir /g, " "],
-  [/ circle /g, " "],
-  [/ ct /g, " "],
-  [/ court /g, " "],
-  [/ cor /g, " "],
-  [/ corner /g, " "],
-  [/ (cmn|common|commons) /g, " "],
-  [/ ctr /g, " "],
-  [/ center /g, " "],
-  [/ pl /g, " "],
-  [/ place /g, " "],
-  [/ plz /g, " "],
-  [/ plaza /g, " "],
-  [/ pkw?y /g, " "],
-  [/ parkway /g, " "],
-  [/ cswy /g, " "],
-  [/ causeway /g, " "],
-  [/ byp /g, " "],
-  [/ bypass /g, " "],
-  [/ mall /g, " "],
-  [/ (xing|crssng) /g, " "],
-  [/ crossing /g, " "],
-  [/ sq /g, " "],
-  [/ square /g, " "],
-  [/ trl? /g, " "],
-  [/ trail /g, " "],
-  [/ (twp|twsp|townsh(ip)?) /g, " "],
-  [/ est(ate)? /g, " estates "],
-  [/ vlg /g, " "], // village
-  [/ village /g, " "],
-  [/ (ste|suite|unit|apt|apartment) #?(\d+) /g, " $1 "],
-  [/ #?(\d+) /g, " $1 "],
-  [/ (&|and) /g, " "],
-];
-
-/**
- * Simplify a text string (especially an address) as much as possible so that
- * it might match with a similar string from another source.
- * @param {string} text
- * @returns {string}
- */
-function matchable(text) {
-  return text
-    .toLowerCase()
-    .replace(POSSESSIVE_PATTERN, " ")
-    .replace(PUNCTUATION_PATTERN, " ")
-    .replace(MULTIPLE_SPACE_PATTERN, " ")
-    .trim();
-}
-
-function matchableAddress(text, line = null) {
-  let lines = Array.isArray(text)
-    ? text
-    : text.split(ADDRESS_LINE_DELIMITER_PATTERN);
-
-  // If there are multiple lines and it looks like the first line is the name
-  // of a place (rather than the street), drop the first line.
-  if (lines.length > 1 && !/\d/.test(lines[0])) {
-    lines = lines.slice(1);
-  }
-
-  if (line != null) {
-    lines = lines.slice(line, line + 1);
-  }
+export const db = createDbClient("Scripts");
```

Comment:
🎉 

---

Side note: it’s still nice to use a separate client instance here rather than the global one from `src/db.ts` because the `main()` function will close the connection. The lifecycle expectations for the two instances are a little different, so keeping them separate makes maintaining tests easier.